website/content/docs/_index.md

```diff
@@ -2,8 +2,129 @@
 title= "Wash Documentation"
 +++
 
-You can find Wash documentation here. It's a work in progress!
+* [Wash Commands](#wash-commands)
+  * [wash](#wash)
+  * [wash clear](#wash-clear)
+  * [wash exec](#wash-exec)
+  * [wash find](#wash-find)
+  * [wash history](#wash-history)
+  * [wash info](#wash-info)
+  * [wash list](#wash-list-ls)
+  * [wash meta](#wash-meta)
+  * [wash ps](#wash-ps)
+  * [wash server](#wash-server)
+  * [wash tail](#wash-tail)
+* [Core Plugins](#core-plugins)
+  * [AWS](#aws)
+  * [Docker](#docker)
+  * [Kubernetes](#kubernetes)
+* [Plugin Concepts](#plugin-concepts)
+  * [➠External plugins]
+  * [➠Go Plugins]
+  * [➠Server API]
 
-* [External plugins](/wash/docs/external_plugins)
-* [Go Plugins](/wash/docs/go_plugins)
-* [Server API](/wash/docs/api)
\ No newline at end of file
+## Wash Commands
+
+Wash commands aim to be well-documented in the tool. Try `wash help` and `wash help <subcommand>` for specific options.
+
+Most commands operate on Wash resources, which are addressed by their path in the filesystem.
+
+### wash
+
+The `wash` command can be invoked on its own to enter a Wash shell.
+
+Invoking `wash` starts the daemon as part of the process, then enters your current system shell with shortcuts configured for wash subcommands. All the [`wash server`](#wash-server) settings are also supported with `wash`.
+
+### wash clear
+
+Wash caches most operations. If the resource you're querying appears out-of-date, use this subcommand to reset the cache for resources at or contained within the specified path. Defaults to the current directory if a path is not specified.
+
+### wash exec
+
+For a Wash resource that implements the ability to execute a command, run the specified command and arguments. The results will be forwarded from the target on stdout, stderr, and exit code.
+
+### wash find
```

Comment:
I'd hate updating `wash find` and having to remember to update this too. Is there a way we can automate some of this cruft via go generate or via something in Hugo? It'd be nice if whatever we do sources this stuff from the command objects.